main/nav_backplane_viewer.py

```
@@ -0,0 +1,1365 @@
+def _rad_to_deg_if_units(name: str, units: str | None, arr: np.ndarray) -> np.ndarray:
+    if units and units.lower() == 'rad':
+        return cast(np.ndarray, np.degrees(arr))
+    # Heuristic: some HDU names encode angle but BUNIT may be missing
+    lower = name.lower()
+    if any(k in lower for k in ('longitude', 'latitude', 'incidence', 'emission', 'phase')):
+        return cast(np.ndarray, np.degrees(arr))
+    return arr
+
+
+def _absolute_range_for(name: str,
+                        units: str | None,
```

Ruff (0.14.5)

Unused function argument: `units`

(ARG001)